### DISCUSSION TITLE
Feature request: Current Realtime connections in Prometheus endpoint

### DISCUSSION THREAD

**Discussion by toystorynova** [supabase/supabase#7272] [Feature Requests]
Adding the current realtime connections to Metrics endpoint (customer/v1/privileged/metrics) would be a nice addition. 

`SELECT COUNT(*) FROM realtime.subscription`

I don't believe this is there either but the realtime connection requests number shown in the fourth panel on the Dashboard would be great to have.

**Comment by w3b6x9** [comment]
@toystorynova thanks for the feedback! will pass this on to the rest of the Realtime team.
